### PR TITLE
Added By base Schedule Sheet Modals and Components

### PR DESCRIPTION
**Pull request by leekimah**

### CODE REVIEW

File: apps/employee-monitoring/src/components/inputs/SelectList.tsx [hunk 1]
```diff
@@ -166,7 +166,7 @@ export function MySelectList({
         <ul
           className={`${
             isOpen ? 'block' : 'hidden'
-          } border absolute  rounded  max-h-[12em] bg-white z-50 overflow-y-auto w-full left-0 top-[calc(100%+.25em)] `}
+          } border absolute  rounded  max-h-[12em] bg-whibte z-50 overflow-y-auto w-full left-0 top-[calc(100%+.25em)] `}
```
Comment (dev-gscwd) [inline apps/employee-monitoring/src/components/inputs/SelectList.tsx:169]: Correct spelling

File: apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+                </div>
+              </Modal.Footer>
+            </Modal>
+
+            <SelectFieldSchedSsModal
+              modalState={selectScheduleModalIsOpen}
+              setModalState={setSelectScheduleModalIsopen}
+              closeModalAction={closeSelectScheduleModal}
+            />
+            <form id="AddFieldSsForm">
```
Comment (dev-gscwd) [inline apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx:148]: addFieldSsForm

File: apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+              className="px-3 py-2 text-gray-700 bg-gray-200 rounded text-md hover:bg-gray-300"
+              onClick={closeModalAction}
+            >
+              Cancel
+            </button>
+
+            <button
+              className="px-3 py-2 text-white bg-blue-500 rounded text-md disabled:cursor-not-allowed hover:bg-blue-400"
+              type="submit"
+              form="AddFieldGsModal"
```
Comment (dev-gscwd) [inline apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx:292]: addFieldGsModal

**Review by dev-gscwd — APPROVED** [review]

File: apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx (hunk 2 above)
Comment (leekimah) [reply apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx]: Changed it to addFieldSsForm and removed the extra form

File: apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx (hunk 3 above)
Comment (leekimah) [reply apps/employee-monitoring/src/components/sidebar-items/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx]: replaced with addFieldSsModal

**Pull request merged**